backup/nextflow.config

```diff
@@ -0,0 +1,192 @@
+/*
+ * -------------------------------------------------
+ *  nf-core/rnavar Nextflow config file
+ * -------------------------------------------------
+ * Default config options for all environments.
+ */
+
+// Global default params, used in configs
+params {
+
+  // Workflow flags
+  // TODO nf-core: Specify your pipeline's command line flags
+  genome = false
+  input = null
+  input_paths = null
+  single_end = false
+  outdir = './results'
+  publish_dir_mode = 'copy'
+
+  // Boilerplate options
+  multiqc_config = false
+  email = false
+  email_on_fail = false
+  max_multiqc_email_size = 25.MB
+  plaintext_email = false
+  monochrome_logs = false
+  help = false
+  igenomes_base = 's3://ngi-igenomes/igenomes'
+  tracedir = "${params.outdir}/pipeline_info"
+  igenomes_ignore = false
+  custom_config_version = 'master'
+  custom_config_base = "https://raw.githubusercontent.com/nf-core/configs/${params.custom_config_version}"
+  hostnames = false
+  config_profile_name = null
+  config_profile_description = false
+  config_profile_contact = false
+  config_profile_url = false
+  validate_params = true
+  show_hidden_params = false
+  schema_ignore_params = 'genomes,input_paths'
+
+  // Defaults only, expecting to be overwritten
+  max_memory = 128.GB
+  max_cpus = 16
+  max_time = 240.h
+
+}
+
+// Container slug. Stable releases should specify release tag!
+// Developmental code should specify :dev
+process.container = 'nfcore/rnavar:dev'
```

Comment:
```suggestion
```
We won't need a main container for this pipeline as we're going to have a container per process approach with DSL2